src/iv/imageviewer.cpp

```diff
@@ -386,6 +386,17 @@ ImageViewer::createActions()
     //    toggleImageAct->setEnabled(true);
     connect(toggleImageAct, SIGNAL(triggered()), this, SLOT(toggleImage()));
 
+    showDataWindowAct = new QAction(tr("Show Data Window"), this);
+    showDataWindowAct->setCheckable(true);
+    connect(showDataWindowAct, SIGNAL(triggered()), this,
+            SLOT(toggleDataWindow()));
+
+    showDisplayWindowAct = new QAction(tr("Show Display Window"), this);
+    showDisplayWindowAct->setCheckable(true);
+    connect(showDisplayWindowAct, SIGNAL(triggered()), this,
+            SLOT(toggleDisplayWindow()));
+    showDisplayWindowAct->setChecked(true);  // Show display window by default
```

Comment:
I think maybe I prefer that the default be to not draw either one, and you can toggle it on if you want to see where they are? But I'm willing to be talked out of that if you had a reason to show it by default.

Reply:
I don't have a strong preference for it, it was mostly to follow the description in https://github.com/AcademySoftwareFoundation/OpenImageIO/issues/308. Happy to disable them by default.

Reply:
Oh boy, after 12 years, I didn't realize it was me who made the request! (Though I was probably transcribing what somebody told me.)

I'm fine with the dash/solid choice, we can easily change that later if people object after trying it.

I think if we guaranteed that the lines were *just outside* the window in question, and thus the default view of showing exactly the display window would not show it, I would be ok with defaulting display window guide to be on. But since we can't control its exact placement, I believe I prefer it to be off so that it doesn't obscure any pixels or interfere with the view of the image, by default. So let's default to off and people can turn on when they want to check where the windows are.

Reply:
No problem, that sounds fair to me. I can disable them both by default.